enhancements/alert-on-operator-updates.md

```diff
@@ -0,0 +1,158 @@
+---
+title: alert-on-operator-updates
+authors:
+  - "@harishsurf"
+reviewers:
+  - 
+approvers:
+  - 
+creation-date: 2020-06-10
+last-updated: 2020-06-10
+status: POC
+see-also:
+  - "N/A"  
+replaces:
+  - "N/A"
+superseded-by:
+  - "N/A"
+---
+
+# OLM alerting/rejecting dangerous Operator updates
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `POC`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Description
+
+[Operator Lifecycle Manager (OLM) project](https://github.com/operator-framework/operator-lifecycle-manager) 
+oversees installation, updates, and management of the lifecycle of all of the Operators (and their associated 
+services) running across a Kubernetes cluster. Given that the installed Operators have the freedom to do almost anything with appropriate access privileges, there is a 
+potential threat that these actions may cause harm to the cluster, are malicious, or detrimental to a cluster upgrade. 
+
+This enhancement deals with handling two such Operator actions:  
+- Alert is triggered when the operator deploys operands with an invalid [Pod Disruption Budget (PDB)](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget) that would cause an openshift upgrade issue
+- Alert is triggered when an operator tries to modify an existing on-board [Security Context Constraints (SCCs)](https://www.openshift.com/blog/understanding-service-accounts-sccs))
+ 
+
+### Goals
+
+1. Provide a mechanism to alert when an invalid PDB is being applied on to a cluster 
+2. Prevent operators from modifying existing SCCs in the cluster
+3. Provide a configuration option to cluster admins for a permissive mode vs. a enforcing mode
```

Comment:
Can you be more specific here? If this is an alerting mode for a set of rules, is one of the goals also to define a concept for generic alerting rules, which PDB's and SCC's are examples of?